wiki/en/Running-a-Server.md

```diff
@@ -155,71 +155,71 @@ To register with one of the Directories built into the Jamulus Client, replace `
 
-##### `--serverpublicip` 
+##### `--serverpublicip`
 The public IP address of the Server if connecting to a Directory behind the same NAT. See [the Directories guide](Directories) for further information.
 
-##### `--directoryfile` 
+##### `--directoryfile`
 _Directories only:_ Remember registered Servers even if the Directory is restarted. See [the Directories guide](Directories) for further information.
 
-##### `-f or --listfilter` 
+##### `-f or --listfilter`
 _Directories only:_ Allowlist Servers registering on the Server. See [the Directories guide](Directories) for further information.
 
 ---
 
 ### General Server options
-These options can be used regardless of which mode your server is running in (although some may not be relevant for Directories).
+These options can be used regardless of which mode your Server is running in (although some may not be relevant for Directories).
 
-##### `-d or --discononquit` 
+##### `-d or --discononquit`
 Normally, when a Server is stopped or restarted, any Clients that have not pressed their **Disconnect** buttons will automatically re-establish the connection when the Server returns. This option forces Clients to manually re-establish their connections to the Server in this scenario.
 
-##### `-F or --fastupdate` 
+##### `-F or --fastupdate`
 Reduces latency if Clients connect with **Enable Small Network Buffers** option. Requires faster CPU to avoid dropouts, and more bandwidth to enabled Clients.
 
-##### `-l or --log` 
+##### `-l or --log`
 Enable logging, set path and file name
 
-##### `-m or --htmlstatus` 
+##### `-m or --htmlstatus`
 Enable HTML status file, set path and file name
 
 **Note:** This feature is deprecated, and may disappear in a future release.
 
-##### `-P or --delaypan` 
+##### `-P or --delaypan`
 Start with delay panning enabled. This option uses small differences in sound arrival time between the two ears. It produces a stereo effect similar to natural human hearing when compared to normal “volume” panning.
 
 ##### `-s` or `--server`
 Start Jamulus in Server mode
 
-##### `--serverbindip` 
-Specify the IP address the Jamulus process will bind to. 
+##### `--serverbindip`
+Specify the IP address the Jamulus process will bind to.
 
 Normally, Jamulus will listen on all IP addresses on the host machine.  Where the host has multiple network addresses, this option allows one of the addresses to be chosen.
 
-##### `-T or --multithreading` 
+##### `-T or --multithreading`
 Use multithreading to make better use of multi-core CPUs. This setting may help the Server support more Clients. See also `--numchannels`
 
-##### `-u or --numchannels` 
+##### `-u or --numchannels`
 Maximum number of channels (Clients)
 
-##### `-z or --startminimized` 
-Start the Jamulus Server graphical user interface in the minimized window state.
+##### `-z or --startminimized`
+Start the Jamulus Server graphical user interface in the minimised window state.
```

Comment:
```suggestion
Start the Jamulus Server graphical user interface in the minimized window state.
```

Reply:
This is a bit of an anomaly - we decided to [use British spelling](https://jamulus.io/contribute/Style-and-Tone#we-use-british-english) a while ago, but obviously this option is the US one :-)